src/flake8_bugbear/plugins/cached_instance_method.rs

```diff
@@ -0,0 +1,31 @@
+use rustpython_ast::Expr;
+
+use crate::ast::helpers::compose_call_path;
+use crate::ast::types::{Range, ScopeKind};
+use crate::check_ast::Checker;
+use crate::checks::{Check, CheckKind};
+
+const CACHE_FUNCTIONS: [&str; 4] = [
+    "functools.lru_cache",
+    "functools.cache",
+    "lru_cache",
+    "cache",
+];
```

Comment:
Do you mind tweaking this to use the `checker.from_imports` to verify that these are actually imported from `functools`?

Reply:
(There are some TODOs to do this in other files too, like `src/flake8_bugbear/plugins/mutable_argument_default.rs`, but it's a newer feature.)

Reply:
Got it. Do we have an example?

Reply:
`flake8_2020/plugins.rs`

Reply:
Done!